frontend_app_agenda/src/container/Agenda.jsx

```diff
@@ -193,7 +191,8 @@ class Agenda extends React.Component {
           idWorkspace: a.agenda_type === 'private' ? '' : a.workspace_id
         }))
       },
-      userLang: state.loggedUser.lang
+      userLang: state.loggedUser.lang,
+      shouldDisplayCaldavzapSidebar: !(state.userWorkspaceList.length === 1 && state.userWorkspaceList[0].agenda_type === 'workspace')
```

Comment:
My point is showing/hidding the calendar list is based on the uri.

- If you are in the `/ui/agendas/` then you show the sidebar,
- if you are in `/workspaces/<id>/agenda` you hide the agenda list

In a near future, you will have the case where the calendar is out of tracim calendars, in this case we will have to show the list

Reply:
I've changed it so that it's now the frontend core that tells whether or caldavzap should force display the sidebar regarding the route.

Caldavzap algo is now:
if frontend core set `forceShowSidebar: true` then always show sidebar.
if frontend core set `forceShowSidebar: false` then show sidebar **only** if there is more than one calendar to display

see https://github.com/tracim/tracim/pull/1601/files#diff-eaa4f71493ea0798083dc4ef3d2e60e9R1995

Reply:
A remark by the way: `:idws` means id workspace, I think... in English in must be wsid (workspace id). You should write all your stuff in English.